docs/third-party-developers/extensibility/data-store/collections.md

```diff
@@ -122,6 +126,86 @@ or
     -   _query_ `Object`: The query arguments for the collection, eg. `{ order: 'ASC', sortBy: Price }`.
     -   _ids_ `Array`: If the collection route has placeholders for ids you provide the values for those placeholders in this array (in order).
 
+### getFromState
+
+This selector will return the state from the collections store.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `object`: The state from the collections storew ith the following properties:
+   	- _namespace_ `string`: The route namespace for the collection, eg. `/wc/blocks`.
+    - _resourceName_ `string`: The resource name for the collection, eg. `products/attributes`.
+    - _query_ `object`: The query arguments for the collection, eg. `{ order: 'ASC', sortBy: Price }`.
+    - _ids_ `array`: If the collection route has placeholders for ids you provide the values for those placeholders in this array (in order).
+   	- _type_ `string`: type of the collections ie `items`.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const store = select( 'wc/store/collections' );
+const state = store.getFromState();
+```
+
+### getCollectionHeaders
+
+This selector will return the headers for a collection.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `object`: The headers for the collection with the following keys:
+   	-   _namespace_ `string`: The route namespace for the collection, eg. `/wc/blocks`.
+   	-   _resourceName_ `string`: The resource name for the collection, eg. `products/attributes`.
+   	-   _query_ `object`: The query arguments for the collection, eg. `{ order: 'ASC', sortBy: Price }`.
+   	-   _ids_ `array`: If the collection route has placeholders for ids you provide the values for those placeholders in this array (in order).
+   	-   _type_ `string`: `headers` for this selector.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const store = select( 'wc/store/collections' );
+const headers = store.getCollectionHeaders(state, namespace, resourceName, queryString);
+```
+
+### getCollectionError
+
+This selector will return any error that occurred while fetching a collection.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   _error_ `object`: The error that occurred while fetching the collection with the following properties:
```

Comment:
The functions above show
```js
`object`: [...]
```
while this function shows
```js
_error_ `object`: [...]
```

I suggest leaving `_error_` out to be consistent with the other docs.